site/content/resources/job-description-templates/marketing/junior-copy-writer-job-description-template.md

```diff
@@ -1,30 +1,34 @@
 ---
 type: jobs
 Title: Junior Copy Writer Job Description Template
+aliases:
+  - /resources/junior-copy-writer-job-description-templates
```

Comment:
```suggestion
  - /resources/junior-copy-writer-job-description-template
```
in order to match https://enlist.io/resources/junior-copy-writer-job-description-template